engine/lite/embeddedDeploymentManager/src/main/scala/pl/touk/nussknacker/streaming/embedded/EmbeddedProcessStateDefinitionManager.scala

```diff
@@ -0,0 +1,40 @@
+package pl.touk.nussknacker.streaming.embedded
+
+import pl.touk.nussknacker.engine.api.deployment.ProcessActionType.ProcessActionType
+import pl.touk.nussknacker.engine.api.deployment.simple.SimpleProcessStateDefinitionManager
+import pl.touk.nussknacker.engine.api.deployment.{ProcessActionType, ProcessStateDefinitionManager, StateStatus}
+
+import java.net.URI
+
+object EmbeddedProcessStateDefinitionManager extends ProcessStateDefinitionManager  {
```

Comment:
Maybe we can reduce amount of boilerplate? By having sth like `OverridingProcessStateDefinitionManager` where you pass those maps in constructor? And use it also for Flink?